packages/chat/src/components/ColorTheme/index.ts

```diff
@@ -0,0 +1 @@
+export { ColorTheme } from './ColorTheme.component';
```

Comment:
we don't need this file..  
(I saw that on the ui library there is an index.ts file in each dir, but it's not needed. we can just export what we need in the files themselves. It only makes sense to have them when exporting specifically for another library to use - like in our case, the 'ui.ts' or 'stories.ts' files)